src/Microsoft.Health.Fhir.Core/Features/Definition/ISearchParameterDefinitionManager.cs

```diff
@@ -13,7 +14,7 @@ namespace Microsoft.Health.Fhir.Core.Features.Definition
     /// <summary>
     /// Provides mechanism to access search parameter definition.
     /// </summary>
-    public interface ISearchParameterDefinitionManager
+    public interface ISearchParameterDefinitionManager : IStartable
```

Comment:
I don't think the interface ISearchParameterDefinitionManager should build on IStartable, these seem like unrelated concepts

Reply:
Let me know what you think of 138de6b. I removed `IStartable` from the interface but exposed the `Start()` method so that it could be called from `SqlServerFhirModel`.

Reply:
As discussed, I removed `Start()` from `ISearchParameterDefinitionManager` in b2f049a.